test/index.test.js

```diff
@@ -0,0 +1,107 @@
+const { map } = require('../lib/index');
```

Comment:
```suggestion
const { map, filter, indexOf, reduce, every } = require('../lib/index');
```
You can have a single import line like above :)